src/components/Login/ForgotPassword.jsx

```diff
@@ -35,42 +60,85 @@ const ForgotPassword = React.memo(() => {
     }
   };
 
+  const handleInput = e => {
+    const { name, value } = e.target;
+    let errorData = { ...message };
+    //use schema.extract instead if joi package is updated
+    var vaildateResult = {};
```

Comment:
there's a misspelling on the "validateResult" variable, which is being spelled as "vaildateResult"